### PR TITLE
Feature/ PDF handler

### PR DESCRIPTION
**Pull request by coppee**
Need to discuss what we do with the PDF (store, send, upload) and how to config that

### CODE REVIEW

**Review by Davidmattei — DISMISSED** [review]
Great!

File: Handler/PdfHandler.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    public function handle(SubmissionConfig $submission, FormInterface $form, FormConfig $config, AbstractResponse $previousResponse = null): AbstractResponse
+    {
+        try {
+            $renderedSubmission = $this->renderer->render($submission, $form, $config, $previousResponse);
+            $pdfConfig = new PdfConfig($renderedSubmission);
+
+            $stream = $this->pdfGenerator->getStreamedResponse($pdfConfig->getBody());
+            dd($stream);
```
Comment (dameert) [inline Handler/PdfHandler.php:39]: @Davidmattei you accepted this! :p

File: Handler/PdfHandler.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+use EMS\FormBundle\FormConfig\SubmissionConfig;
+use EMS\FormBundle\Submit\AbstractResponse;
+use EMS\FormBundle\Submit\FailedResponse;
+use EMS\SubmissionBundle\FormConfig\PdfConfig;
+use EMS\SubmissionBundle\Service\DirectoryManager;
+use EMS\SubmissionBundle\Service\SubmissionRenderer;
+use EMS\SubmissionBundle\Submit\PdfResponse;
+use Symfony\Component\Form\FormInterface;
+
+final class PdfHandler extends AbstractHandler
```
Comment (dameert) [inline Handler/PdfHandler.php:19]:
```suggestion
final class PdfCreateHandler extends AbstractHandler
```

Comment (dameert) [reply Handler/PdfHandler.php:19]: A second handler is needed for the use case "save on sftp server" --> SftpStorageHandler

Comment (dameert) [reply Handler/PdfHandler.php:19]: SftpStorageHandler requires that previousResponse is not null, and implements a new interface to indicate that we have a "saveable" file.

**Review by dameert — CHANGES_REQUESTED** [review]

**Comment by Davidmattei** [comment]
Closed added in version 2.0 https://github.com/ems-project/EMSSubmissionBundle/pull/31

**Pull request closed without merging**